### PR TITLE
shellcheck: refresh

### PR DESCRIPTION
**Pull request by bl-ue**

### CODE REVIEW

File: pages/common/shellcheck.md [hunk 1]
```diff
@@ -1,20 +1,33 @@
...
 - Check a shell script:
 
-`shellcheck {{file.sh}}`
+`shellcheck {{path/to/script.sh}}`
 
-- Override script's shebang:
+- Check a shell script interpreting it as the specified shell dialect (overrides the shebang at the top of the script):
 
-`shellcheck --shell {{sh|bash|ksh}} {{file.sh}}`
+`shellcheck --shell {{sh|bash|dash|ksh}} {{path/to/file.sh}}`
```
Comment (einverne) [inline pages/common/shellcheck.md:13]: To keep consistent, should it be

```suggestion
`shellcheck --shell {{sh|bash|dash|ksh}} {{path/to/script.sh}}`
```

File: pages/common/shellcheck.md [hunk 2]
```diff
@@ -1,20 +1,33 @@
...
 
-- Ignore certain errors:
+- Ignore one or more error types:
 
-`shellcheck --exclude {{SC1009}} {{file.sh}}`
+`shellcheck --exclude {{SC1009,SC1073}} {{path/to/file.sh}}`
```
Comment (einverne) [inline pages/common/shellcheck.md:17]: Same here.

**Review by einverne — COMMENTED** [review]
A little suggestion to keep consistent.

File: pages/common/shellcheck.md (hunk 2 above)
Comment (Waples) [reply pages/common/shellcheck.md:17]: consistency, `path/to/script.sh` or `path/to/file.sh` @einverne , in your other comment you say script :P

File: pages/common/shellcheck.md (hunk 1 above)
Comment (navarroaxel) [reply pages/common/shellcheck.md:13]: Do you use `Hey dude, run this sh script`? I say `sh file` and `bash script` so I vote for `file.sh` :rofl:

Comment (bl-ue) [reply pages/common/shellcheck.md:13]: @einverne you're right—good catch.

You think we should use `file.sh`?  I'm fine with that—we should also change it in #5714, as well, then. @sbrl

Comment (sbrl) [reply pages/common/shellcheck.md:13]: It really doesn't matter. I suggest using whichever is more clear on a case-by-case basis - this isn't something that needs standardising or worrying too much over consistency.

**Review by einverne — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**